kcidb/__init__.py

```diff
@@ -252,11 +252,26 @@ def query_main():
 def schema_main():
     """Execute the kcidb-schema command-line tool"""
     sys.excepthook = misc.log_and_print_excepthook
-    description = 'kcidb-schema - Output latest I/O JSON schema'
+    description = 'kcidb-schema - Output latest or older I/O JSON schema'
     parser = misc.OutputArgumentParser(description=description)
+    version = io.schema.LATEST
+    parser.add_argument(
+        'schema_version',
+        metavar='Schema Version',
+        type=int,
+        help=f'KCIDB schema version. Default is {version.major}',
```

Comment:
Could you be more specific about what the schema version would be used for, here? Also, using `KCIDB` here is a little redundant. Finally, it would be good to mention that the default is always the latest version, so people don't have to wonder what would happen after an upgrade.
Overall, I think this would be better:
```suggestion
        help=f'Major version number of the schema to output. '
        'Default is the latest version (currently {version.major})',
```